src/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if (WIN32)
                        ${DLL_LOCATION} ${UNIT_TEST_DIR})
 else(WIN32)
     find_package(LibXml2 2.9 REQUIRED)
-    find_package(CURL 7.31 REQUIRED)
+    find_package(CURL 7.29 REQUIRED)
```

Comment:
It might be better for you to fork the repo and make any version specific changes that you need to support the version of CentOS you're running on.